### PR TITLE
Refactor template macros

### LINKED ISSUE #206: Refactor templace macros
- [ ] Move macros from `repologyapp/templates/macros.html` into dedicated files (`repologyapp/templates/macros/`)
- [ ] Convert all imports to `from macros/XXX import YYY`

The example of how it's done right is `macros/link_status.html`

### PR DESCRIPTION
**Pull request by tocic**
Closes #206.

### CODE REVIEW

File: repologyapp/templates/admin/cpes.html [hunk 1]
```diff
@@ -56,7 +56,7 @@
 	{% for cpe in cpes %}
 		<tr>
 			<td class="minimal-column"><a href="{{ url_for('project_versions', name=cpe.effname) }}">{{ cpe.effname }}</a></td>
-			<td class="minimal-column">{{ mcpes.panels(cpe) }}</td>
+			<td class="minimal-column">{{ panels(cpe) }}</td>
```
Comment (AMDmi3) [inline repologyapp/templates/admin/cpes.html:59]: `panels` is too ambiguous. Please import `panels as cpe_panels` or `import "_macros/cpes.html as cpes` and invoke as `cpe_panels` or `cpe.panels`.

**Review by AMDmi3 — CHANGES_REQUESTED** [review]

File: repologyapp/templates/admin/cpes.html (hunk 1 above)
Comment (tocic) [reply repologyapp/templates/admin/cpes.html:59]: I agree that it's ambiguous. But maybe it's better to rename ambiguous entries instead? That way we can consistently use `from _macros/XXX import YYY` everywhere.

What do you think about these names?

`meta` -> `enable_autorefresh`
`toggler` -> `autorefresh_toggler`
`panels` -> `cpe_components`

**Comment by AMDmi3** [comment]
I don't consider 'inconsistent' imports as a problem. It's just a tool similar to python imports which allows you to import just what you need under descriptive names. But your proposal looks good too.

**Comment by tocic** [comment]
@AMDmi3 I've fixed the requested lines (in case you didn't get notified).

**Pull request merged**